src/config.ts

```diff
@@ -1,14 +1,9 @@
-import { ConnectionOptions } from 'typeorm/browser'
-import { entityList } from './lib/storage/entities'
+import * as typeOrmConf from '../ormconfig'
 
-const typeOrmConf: ConnectionOptions = {
-  type: 'react-native',
-  database: 'LocalSmartWalletData',
-  location: 'default',
-  logging: ['error', 'query', 'schema'],
-  synchronize: true,
-  entities: entityList,
-}
+// typeorm CLI is not able to connect to the sqlite db on the phone,
```

Comment:
Doesn't this comment belong in the `ormconfig.ts` file instead?

Reply:
Probably in both places :grin: